instrumentation/aerospike-client/aerospike-client-7.1/library/src/main/java/io/opentelemetry/instrumentation/aerospike/v7_1/AerospikeMetricsAdvice.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.aerospike.v7_1;
+
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.metrics.DoubleHistogramBuilder;
+import io.opentelemetry.api.metrics.LongCounterBuilder;
+import io.opentelemetry.api.metrics.LongUpDownCounterBuilder;
+import io.opentelemetry.extension.incubator.metrics.ExtendedDoubleHistogramBuilder;
+import io.opentelemetry.extension.incubator.metrics.ExtendedLongCounterBuilder;
+import io.opentelemetry.extension.incubator.metrics.ExtendedLongUpDownCounterBuilder;
+import io.opentelemetry.instrumentation.api.internal.SemconvStability;
+import io.opentelemetry.semconv.SemanticAttributes;
+import java.util.ArrayList;
+import java.util.List;
+
+final class AerospikeMetricsAdvice {
+  private AerospikeMetricsAdvice() {}
+
+  @SuppressWarnings("deprecation") // until old http semconv are dropped in 2.0
```

Comment:
old http semconv has been removed from the agent, you can also remove it

Reply:
Alright, removed it.